### PR TITLE
E-1196 fix: consent provider text update

### PR DESCRIPTION
**Pull request by lamartire**
@ifiri ready for review

### CODE REVIEW

File: tests/unit/specs/components/forms/Scopes.spec.js [hunk 1]
```diff
@@ -23,30 +34,39 @@ describe('Scopes', () => {
     });
 
     it('should render given scopes', () => {
-      wrapper.setProps({
-        scopesList: ['foo', 'bar'],
+      wrapper = wapperFactory({
```
Comment (ifiri) [inline tests/unit/specs/components/forms/Scopes.spec.js:37]: add please `expect.assertions`

File: src/locales/i18n.en.json [hunk 2]
```diff
@@ -126,7 +126,7 @@
       "title": "Change password"
     },
     "scopes": {
-      "allowScopes": "This application is requesting permission to access the following details from you. Press \"Allow\" if you would like to grant access to this information.",
+      "allowScopes": "{appName} application is requesting permission to access the following details from you. Press \"Allow\" if you would like to grant access to this information.",
```
[the remark this answers is not in the document]
Comment (lamartire) [reply src/locales/i18n.en.json:129]: I did it as it was requested in task description

File: tests/unit/specs/components/forms/Scopes.spec.js (hunk 1 above)
Comment (lamartire) [reply tests/unit/specs/components/forms/Scopes.spec.js:41]: done

File: src/locales/i18n.en.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (ifiri) [reply src/locales/i18n.en.json:129]: then ok

File: src/components/screens/public/ConsentProvider/ConsentProviderController.js [hunk 3]
```diff
@@ -68,6 +68,7 @@ class ConsentProviderController extends VuexModule {
       requested_scope: scopesList,
       skip: isSkip,
       redirect_url: redirectUrl,
+      client_name: appName,
```
Comment (dudiq) [inline src/components/screens/public/ConsentProvider/ConsentProviderController.js:71]: image

no any app name in description.
structure from server have other fields, you should get `client` at first, then get `client_name`
```
    const {
      requested_scope: scopesList,
      skip: isSkip,
      redirect_url: redirectUrl,
      client,
    } = consentDetails;
    const { client_name: appName } = client;
```
and after that you will receive this

image

and i agree with Danil to make maybe strong/bold app name here, for underline that this is the app name

**Comment by dudiq** [comment]
@ifiri check for approve please

**Comment by ifiri** [comment]
@kazurus approved

**Pull request merged**